best-current-practices/Selecting_an_SVID_format.md

```diff
@@ -0,0 +1,112 @@
+
+### 2.1 X.509-SVID
+[X.509-SVIDs](/standards/X509-SVID.md) are short-lived X.509 certificates containing a SPIFFE ID in the Subject Alternative Name. They are issued by the certificate authority (CA) of their SPIFFE trust domain, and used to authenticate during mutual TLS (mTLS) between workloads.
+
+Casting SVID information to an X.509 certificate format also allows for transport-level encryption to be used between peers with mutual TLS support, meaning X.509-SVIDs enable secure communication in transit between workloads that have such capabilities.
+
+### 2.2 JWT-SVID
+[JWT-SVIDs](/standards/JWT-SVID.md) are signed JSON Web Tokens (JWT) containing a SPIFFE ID in the `sub` claim. It is issued and signed by the JWT issuer of the SPIFFE trust domain, and is used to authenticate in the application layer between workloads.
+
+JWT-SVIDs are bearer tokens, and do not enable any kind of encryption or cryptographic protection against impersonation. Various recommendations exist to mitigate against known vulnerabilities of the JWT as an identity token, and should be followed when JWT-SVIDs are used in SPIFFE.
+
+### 2.3 WIT-SVID
+WIT-SVIDs are signed JWTs containing a SPIFFE ID in the `sub` claim and a public key binding, with proof-of-possession metadata regarding the signing key available in a `cnf` claim.
+
+WIT-SVIDs differ from JWT-SVIDs in that they are proof-of-possession tokens (i.e. **not** bearer tokens). This provides enhanced security guarantees about caller provenance, integrity, and non-repudiation while still maintaining the interoperability of using token-based authentication in the application layer.
+
+## 3. Comparison Matrix
+
+|                       | X.509-SVID                                     | JWT-SVID                                       | WIT-SVID                                                   |
+|-----------------------|------------------------------------------------|------------------------------------------------|------------------------------------------------------------|
+| **Primary use**       | Transport-level identity with mutual TLS       | Request/API-level identity with bearer tokens  | Request/API-level identity with proof-of-possession        |
+| **Layer**             | Transport                                      | Application                                    | Application                                                |
+| **Format**            | X.509 certificate                              | Signed JWT                                     | Signed JWT with public key binding                         |
+| **Proof semantics**   | Holder-of-key (private key bound to cert)      | Bearer token                                   | Holder-of-key (private key bound to workload)              |
```

Comment:
"private key bound to cert" vs "private key bound to workload" is confusing. Both keys have the same characteristics IMO.